util/modlog.js

```diff
@@ -54,15 +54,15 @@ module.exports = class ModLog {
             .setDescription([
                 `**Type**: ${this.type}`,
                 `**User**: ${this.user.tag} (${this.user.id})`,
-                `**Reason**: ${this.reason || `Use \`${this.guild.settings.prefix}reason ${this.case} to claim this log.\``}`
+                `**Reason**: ${this.reason || `Use \`${this.guild.configs.prefix}reason ${this.case} to claim this log.\``}`
             ])
             .setFooter(`Case ${this.case}`)
             .setTimestamp();
         return embed;
     }
 
     async getCase() {
-        const modlogs = await this.provider.get('modlogs', this.guild.id);
+        const modlogs = await this.client.providers.get('modlogs', this.guild.id);

```

Comment:
`this.provider` refers to the db Posrednik uses.

https://github.com/kyranet/Posrednik/blob/addc90db8ec10facdbc2d307ec1cd428d4069a51/util/modlog.js#L85-L87